core/src/main/java/org/owasp/dependencycheck/analyzer/CarthageAnalyzer.java

```diff
@@ -0,0 +1,291 @@
+/*
+ * This file is part of dependency-check-core.
+ *
+ * Licensed under the Apache License, Version 2.0 (the "License");
+ * you may not use this file except in compliance with the License.
+ * You may obtain a copy of the License at
+ *
+ *     http://www.apache.org/licenses/LICENSE-2.0
+ *
+ * Unless required by applicable law or agreed to in writing, software
+ * distributed under the License is distributed on an "AS IS" BASIS,
+ * WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+ * See the License for the specific language governing permissions and
+ * limitations under the License.
+ *
+ * Copyright (c) 2016 IBM Corporation. All Rights Reserved.
```

Comment:
Unless you work for IBM - I would suggest:
```suggestion
 * Copyright (c) 2023 The OWASP Foundation. All Rights Reserved.
```

Reply:
Good catch, I just copied over the header from the CocoapodsAnalyzer. Fixed now.